Sprint-1/destructuring/exercise-1/exercise.js

```diff
@@ -3,7 +3,7 @@ const personOne = {
   age: 34,
   favouriteFood: "Spinach",
 };
-
+let {name, age, favouriteFood} = personOne;
 // Update the parameter to this function to make it work.
 // Don't change anything else.
 function introduceYourself(___________________________) {
```

Comment:
I think you've missed the second part of the task:

_Update the parameter of the function introduceYourself to use destructuring on the object that gets passed in._

Where the ____________ is in introduceYourself(__________)  you'll need to add the correct parameters.